proto/dcs.proto

```diff
@@ -25,17 +23,276 @@ message SetUserFlagRequest {
   uint32 value = 2;
 }
 
-message SetUserFlagResponse {
-  bool success = 1;
+message SetUserFlagResponse {}
+
+message StreamEventsRequest {}
+
+message Event {
+  enum ObjectCategory {
+    UNIT = 0;
+    WEAPON = 1;
+    STATIC = 2;
+    SCENERY = 3;
+    BASE = 4;
+    CARGO = 5;
+  }
+
+  enum Coalition {
+    NEUTRAL = 0;
+    RED = 1;
+    BLUE = 2;
+  }
+
+  message Target {
+    ObjectCategory category = 1;
+    oneof target {
+      uint64 id = 2;
+      string name = 3;
+    }
+  }
+
+  message Position {
+    double x = 1;
+    double y = 2; // alt
+    double z = 3; // y
```

Comment:
I think it would be better for clients if they only had to deal with decimal Lat/Lon/Alt in a specific format consistently and we handle the conversion internally in the lua using https://wiki.hoggitworld.com/view/DCS_func_LLtoLO and https://wiki.hoggitworld.com/view/DCS_func_LOtoLL

I also don't think it is possible to convert the internal x, y, z information to lat/lon from outside the DCS environment anyway so this becomes basically mandatory.

My suggestions for `Position` would be

```
message Position {
  double lat = 1; // e.g. 41.0000
  double lon = 2; // e.g 37.0000
  double alt = 3;  / e.g. 1000.00 (meters)
}
```

Reply:
Good idea, updated the PR accordingly 👍